packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1083,7 +1083,7 @@ export default class Meeting extends StatelessWebexPlugin {
   parseMeetingInfo(meetingInfo) {
     // MeetingInfo will be undefined for 1:1 calls
     if (meetingInfo && meetingInfo.body && !(meetingInfo.errors && meetingInfo.errors.length > 0)) {
-      this.convoId = meetingInfo.body.convoId || this.convoId;
+      this.convoUrl = meetingInfo.body.conversationUrl || this.convoUrl;
```

Comment:
In the jsdoc it is defined as `meetingInfo.conversationUrl` but we are looking at `meetingInfo.body.conversationUrl`. Should those be changed in the jsdoc?